### ISSUE TITLE
[idea] Attach to process 

### ISSUE THREAD

**Issue by PascalRL** [VollRahm/NotEnoughHotkeys#16]
Add a function to attach the macro to a specific process, so it can run in the background.
Another thing that would be really cool to have is the ability to loop the macro.

**Comment by VollRahm** [comment]
@PascalRL By "attaching the macro to a specific process", do you mean an option that only triggers the Macro when a certain process is in foreground?

**Comment by PascalRL** [comment]
@VollRahm No, it's meant so the macro can run in the background attached to a single process/program. The macro continues to run in notepad.exe (for example) regardless if you're focused/using a different program for example chrome, so the macro will not send input to "chrome" (the window you're using/focused on) instead it will send input to the selected process "notepad" aka the process that is running in the background. I hope this clarifies it

**Comment by VollRahm** [comment]
@PascalRL Yes, that is possible by using SendMessage, I will work on implementing it the next time I continue development.

**Comment by PascalRL** [comment]
Awesome! I really love the UI btw, will be waiting for the next update!
